developer-guides/quick-start/README.md

```diff
@@ -19,27 +20,19 @@ You may notice build WARNINGs related to _peer dependencies_  or other transitiv
 2. Install meteor
 
     `curl https://install.meteor.com/ | sh`
-
     There is no need to install `node` or `npm`, as meteor already includes them.  Verify by:
-
     `meteor node -v`
-
     `meteor npm -v`
 
 3. Get rocket.chat code
-
-    `git clone https://github.com/RocketChat/Rocket.Chat.git`
+- `git clone https://github.com/RocketChat/Rocket.Chat.git` (if you wish to clone your own fork, replace with your repo link which may look like __*https://github.com/your-user-name/Rocket.Chat.git*__).
```

Comment:
please revert this.   it adds complexity.

Suggest to add here ->
    Get rocket.chat code (or clone from your own fork)

Reply:
ok